assets/xml/objects/object_fall.xml

```diff
@@ -1,31 +1,31 @@
 ﻿<Root>
     <File Name="object_fall" Segment="6">
-        <DList Name="object_fall_DL_000190" Offset="0x190" />
-        <DList Name="object_fall_DL_000198" Offset="0x198" />
-        <DList Name="object_fall_DL_000220" Offset="0x220" />
-        <Texture Name="object_fall_Tex_000260" OutName="tex_000260" Format="i8" Width="16" Height="16" Offset="0x260" />
-        <DList Name="object_fall_DL_000420" Offset="0x420" />
-        <DList Name="object_fall_DL_000428" Offset="0x428" />
-        <DList Name="object_fall_DL_000490" Offset="0x490" />
-        <DList Name="object_fall_DL_000498" Offset="0x498" />
-        <Array Name="object_fall_Vtx_0004C0" Count="209" Offset="0x4C0">
+        <DList Name="gMoonEmptyDL1" Offset="0x190" /> <!-- Was likely going to be used with gMoonDebrisModel1DL before they made all debris use the same material -->
+        <DList Name="gMoonDebrisMaterialDL" Offset="0x198" />
```

Comment:
In what sense is the word being used here?

Reply:
Do you mean the word "material"? It came from here: https://github.com/zeldaret/oot/pull/985
"Material" means "this DL sets the texture and other stuff, but has no geometry" and "model" means "this DL has the geometry".